CronofyCSharpSampleApp/Controllers/EventsController.cs

```diff
@@ -39,12 +41,12 @@ public ActionResult Create(Models.Event newEvent)
 
             if (ModelState.IsValid)
             {
-                CronofyHelper.UpsertEvent(newEvent.EventId, newEvent.CalendarId, newEvent.Summary, newEvent.Description, newEvent.Start, newEvent.End);
+                CronofyHelper.UpsertEvent(newEvent.EventId, newEvent.CalendarId, newEvent.Summary, newEvent.Description, newEvent.Start, newEvent.End, new Cronofy.Location("", newEvent.Latitude, newEvent.Longitude));

```

Comment:
Why don't we allow the location name to be specified, aren't these supposed to be the full kitchen sink examples?